apps/studio/src/common/formUtils.ts

```diff
@@ -219,7 +219,7 @@ export const commonYupValidation = {
     return Yup.date()
       .required("This field is required")
       .min(
-        new Date(minReleaseDate.getTime() - 24 * 60 * 60 * 1000),
```

Comment:
Maybe a custom Yup validation `test` using a conditional on the dates instead of using `min` to validate the date?

Reply:
Maybe a custom Yup validation `test` using a conditional on the dates instead of using the Yup `min` validation to validate the date? That way we could custom test the Yup validation to match the inclusive conditional of the Date Input `min` attribute.

Reply:
Identified as a timezone issue:

Ran console.logs for the code below, looks like the entered release date is using the local time while the minReleaseDate is using UTC/GMT. Since it's a 2 hour time difference ahead of UTC/GMT on my side the comparison fails for the day of.
1707256800000 (My Time)  => 1707264000000 (Min Release Date)  which returns false.

```
.test("is-valid-releaseDate", "Invalid release date", (value) => {
        const enteredReleaseDate = new Date(value);
        return enteredReleaseDate.getTime() >= minReleaseDate.getTime();
      });
```